README.md

```diff
@@ -101,6 +101,21 @@ This project welcomes contributions and suggestions.  Most contributions require
 
 When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
 
+### Quickstart with Git
+
+1. Install [Git](https://git-scm.com/downloads), [NodeJs](https://nodejs.org/en/download/) and [VS Code](https://code.visualstudio.com/Download) if you haven’t ready.
+2. Create a GitHub account if you don’t already have one, and let us know your GitHub username. We need to add you to the team in GitHub.
+3. Clone [OfficeDev/Office-Addin-Scripts](https://github.com/OfficeDev/Office-Addin-Scripts)
+4. Create a fork off of OfficeDev/Office-Addin-Scripts in GitHub
+5. In a command prompt:
+    1. git remote add :username: :fork-repo-url:     (e.g. git remote add :username: https://github.com/:username:/Office-Addin-Scripts.git --- where “:username:” is your GitHub user name)
```

Comment:
I think putting the parameters inside curly braces ( {username} ) or back-ticks ( `username` ) may work better than using colons.